test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -60,4 +65,17 @@ public static Person generateTestPerson() {
             return null;
         }
     }
+    
+    public static void compareFiles(Path path1, Path path2) throws IOException {
```

Comment:
> isSameTextIgnoreLineEndings(): boolean

Or just `compareTextFiles()`, which uses the "text" mode terminology. Common File IO APIs (e.g. C's fopen() or python's open()) allow operating in "text" mode, where the file's bytes are decoded to the programming language's  native string type and line endings are normalized.

> Just return a boolean

I prefer using JUnit's assert methods so that JUnit can give us a clue on how the files differ.

If we are using JUnit's assert methods though, then this method should be called `assertTextFilesEqual()` or something.